include/bluetooth/conn.h

```diff
@@ -711,8 +711,61 @@ struct bt_conn_oob_info {
 	};
 };
 
+#if defined(CONFIG_BT_SMP_PAIRING_ALLOW)
+/** Info structure for generic_pairing_query */
+struct bt_conn_pairing_auth_rec_info {
+	u8_t bonding_flags:2,
+	     mitm:1,
+	     sc:1,
+	     keypress:1,
+	     ct2:1,
+	     rfu:2;
+};
+
+struct bt_conn_pairing_info {
+	u8_t io_capability;
+	u8_t oob_data_flag;
+	struct bt_conn_pairing_auth_rec_info auth_req;
+	u8_t max_enc_key_size;
+	u8_t init_key_dist;
+	u8_t resp_key_dist;
+};
+#endif /* CONFIG_BT_SMP_PAIRING_ALLOW */
+
 /** Authenticated pairing callback structure */
 struct bt_conn_auth_cb {
+#if defined(CONFIG_BT_SMP_PAIRING_ALLOW)
```

Comment:
Can you predefine this in doxygen so that the documentation for this will be included?

Look for `PREDEFINED` is `doc/zephyr.doxyfile.in`

Reply:
I'm a bit in doubt on that one. I've added Doxygen comments on the structures themselves, but looking at `doc/zephyr/doxyfile.in` it seems that only major flags are present there. Are you sure it should be added there?

Reply:
@demantmrrv Yes, I am sure. It needs to be added there so that it will not excluded from the online documentation.
All #ifdes that are in public header files are enabled there so that all features are included in the documentation.

Reply:
Oh, okay, I wasn't aware of that. I'll add it right away.